index.js

```diff
@@ -162,35 +165,37 @@ exports.init = function (sbot, config) {
     Legacy(sbot, ebt)
   }
 
-  return {
-    replicate: function (opts) {
-      if (opts.version !== 2 && opts.version !== 3) {
-        throw new Error('expected ebt.replicate({version: 3 or 2})')
-      }
-      return toPull.duplex(ebt.createStream(this.id, opts.version, false))
-    },
-    // get replication status for feeds for this id.
-    peerStatus: function (id) {
-      id = id || sbot.id
-      const data = {
-        id: id,
-        seq: ebt.state.clock[id],
-        peers: {}
-      }
-      for (const k in ebt.state.peers) {
-        const peer = ebt.state.peers[k]
-        if (peer.clock[id] != null || peer.replicating[id] != null) {
-          const rep = peer.replicating && peer.replicating[id]
-          data.peers[k] = {
-            seq: peer.clock[id],
-            replicating: rep
-          }
+  function replicate (opts) {
+    if (opts.version !== 2 && opts.version !== 3) {
+      throw new Error('expected ebt.replicate({version: 3 or 2})')
+    }
+    return toPull.duplex(ebt.createStream(this.id, opts.version, false))
+  }
+
+  // get replication status for feeds for this id.
+  function peerStatus (id) {
+    id = id || sbot.id
+    const data = {
+      id: id,
+      seq: ebt.state.clock[id],
+      peers: {}
+    }
+    for (const k in ebt.state.peers) {
+      const peer = ebt.state.peers[k]
+      if (peer.clock[id] !== null || peer.replicating[id] !== null) {
```

Comment:
Probably difficult to see in the diff, but I changed `!=` to `!==`

Reply:
Oops, actually the logic previously was checking for undefined. I'll have to revert the change (we can't use falsy either because `clock[id]` equals `0` is not something we want to catch.